src/followers/followers.service.ts

```diff
@@ -61,6 +66,7 @@ export class FollowersService {
                 id: findFollow.id
             }
         });
-        return unFollowUser;
+        const action = await this.actionsService.addAction('Delete', followedById,'User',findFollow.id,unFollowUser)
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them